### ISSUE TITLE
📖 Product page

### ISSUE THREAD

**Issue by N3v1** [TeamCodeArena/DeliverEase#104]
Hi @SID262000, we need a well-designed product page that is built with HTML, CSS, and JS. 
Maybe I could also help with this.

### Setting projects up
- [x] Create a pages folder where we can store our files for the product page
- [x] Create in the pages folder a style folder where we can store all the CSS files
- [x] Create index.html
- [x] Create style.css

- [ ] Start working on the page
- [ ] When the work is done, deploy it on GitHub Pages

Link to the repo [**here**](https://github.com/TeamCodeArena/deliverease.app)

cc: @Devansh-Singh-Parmar

**Comment by SID262000** [comment]
Yes @N3v1 Please get started, but do remember, we need to get full gear on the development issues in September followed by Testing and Packaging in October, Thanks

**Comment by N3v1** [comment]
Hi @SID262000, Does this repo use any code formatter, if no I could config Prettier

**Comment by SID262000** [comment]
@N3v1 I think it uses Prettier, @AryanGupta0007 Please confirm the same. If it's not present, do add it, Thanks

**Comment by AryanGupta0007** [comment]
@SID262000 i have never heard of prettier. So i am not aware if it's being used or not

**Comment by Devansh-Singh-Parmar** [comment]
I was using Prettier.
cc @SID262000 @N3v1 @AryanGupta0007

**Comment by Atrika10** [comment]
"Hello @N3v1,

I wanted to share that on the frontend side, we use Prettier and Stylelint to maintain code consistency and readability. I kindly request @AryanGupta0007  to confirm the code formatting practices used in the backend.

Thank you for your collaboration.

Best regards,
Atrika

**Comment by KartikKar19** [comment]
@N3v1 Can I help you with this issue in some way??

**Comment by N3v1** [comment]
> @N3v1 Can I help you with this issue in some way??

Sure @KartikKar19, I could use some help to create a page like this https://www.codeedit.app/

**Comment by N3v1** [comment]
We have to start with the pages/html/index.html file

Note: Folder structure for the repo

**pages**
├── html
 │   └── index.html
├── img
├── scripts
 │   └── index.js
└── style
    └── style.css

**Comment by KartikKar19** [comment]
@N3v1 https://kartikkar19.github.io/pages2/ 
This is the basic structure I made according to the site you provided should I work further on this ?
Also, the repo with a folder structure like that is not reading the HTML but only the readme file . Will see if I can fix that and let you know.

**Comment by N3v1** [comment]
Thanks @KartikKar19, maybe we can work with some Animations and background colors

**Comment by N3v1** [comment]
@KartikKar19, How can I help you? Should I create PRs on your fork?

**Comment by KartikKar19** [comment]
@N3v1 yeah sure you can fork it👍

**Comment by N3v1** [comment]
@SID262000, @Devansh-Singh-Parmar, can you create on the TeamCodeArena GitHub profile a new repo called  `deliverease.app` for the product page. Needed for #104

cc: @KartikKar19

**Comment by Devansh-Singh-Parmar** [comment]
Done, provided write access to you both.

cc @N3v1 @KartikKar19

**Comment by SID262000** [comment]
Thanks @Devansh-Singh-Parmar for the follow up. 
Really appreciate all your hard work, @N3v1 @KartikKar19

**Comment by SID262000** [comment]
@KartikKar19 & @N3v1 Please provide an estimate of completion, Thanks!
cc: @Devansh-Singh-Parmar

**Comment by N3v1** [comment]
> @KartikKar19 & @N3v1 Please provide an estimate of completion, Thanks! cc: @Devansh-Singh-Parmar

Maybe we will be done in 2-3 weeks @SID262000 
cc: @Devansh-Singh-Parmar @KartikKar19

**Comment by N3v1** [comment]
To see our progress @SID262000, visit the `deliverease.app` repo

**Comment by SID262000** [comment]
Many Thanks @N3v1 for the prompt update.
I'm not sure if I really have access to the above app, it'll be really great if I could get the access for the same.
cc: @Devansh-Singh-Parmar

**Comment by Devansh-Singh-Parmar** [comment]
that's kind of impossible.

cc @SID262000

**Comment by SID262000** [comment]
Is it regarding the access to the app, @Devansh-Singh-Parmar 
cc: @N3v1

**Comment by Devansh-Singh-Parmar** [comment]
Okay, I misunderstood that, provided access to you.
cc @SID262000 @N3v1

**Comment by N3v1** [comment]
@Maghish Is this issue Hacktoberfest related?

**Comment by Devansh-Singh-Parmar** [comment]
No, thank you.
Cc @N3v1

**Comment by SID262000** [comment]
We can have it as a Hacktoberfest contribution.
cc: @Devansh-Singh-Parmar

**Comment by Maghish** [comment]
Yes @N3v1 This is one of the top priority issues I think. so it's better to add it to the hacktoberfest. Thanks!
cc: @SID262000 @Devansh-Singh-Parmar

**Comment by N3v1** [comment]
I have created the basic structure, but there is still a lot of work to be done on the styling. Additionally, I would like to share the sidemap for the home page, which can be found here:
<img width="100" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/30189d0c-ba8a-402e-993c-ee39828ded3a">

 I have also added our logo and some headings, as shown in the following screenshot:
<img width="1552" alt="Bildschirmfoto 2023-10-10 um 13 50 11" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/b0821dd1-ef1a-480d-943e-f3d2475f78b5">

Lastly, I have included some markups for the design, which can be seen in the following screenshot:
<img width="1439" alt="SCR-20231010-mhzn" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/be6dae79-0409-4dc4-aa52-553a5f80a678">
The description text has to be centered, a navigation bar has to be created and stuff like that so i just began working on it

cc: @Devansh-Singh-Parmar @SID262000

**Comment by Devansh-Singh-Parmar** [comment]
Great, work.
cc @N3v1

**Comment by N3v1** [comment]
But I really need help because I can't do all the pages by myself. Please DM me on Slack or Discord if someone is interested. Note I'm using Next.js with Javascript

Discord username: nh2487

**Comment by Maghish** [comment]
I think integrating Next.js with django is a bit impossible but I will try to find a solution if you all confirm that we are going to use Next.js for front-end. Thanks!
cc: @SID262000 @Devansh-Singh-Parmar @N3v1

**Comment by SID262000** [comment]
@Maghish I wouldn't recommend going ahead with specific frameworks and rather sticking to the foundational JS and TS.
cc: @N3v1

**Comment by Maghish** [comment]
alright @SID262000

**Comment by N3v1** [comment]
> I think integrating Next.js with django is a bit impossible but I will try to find a solution if you all confirm that we are going to use Next.js for front-end. Thanks! cc: @SID262000 @Devansh-Singh-Parmar @N3v1

Hey @Maghish, I apologize for any inconvenience caused. The product page is a separate page and is not related to Django, Python, or the main deliver ease repo. To handle this, we have created another repository in our organization. Additionally, we can use HTML, CSS, and JS to build the page, but we have found that Next.js makes it easier to create visually appealing websites.

cc: @SID262000 @Devansh-Singh-Parmar

**Comment by SID262000** [comment]
@N3v1 If you think you're more comfortable around using Next.js, feel free to proceed accordingly. Thanks!
cc: @Maghish
